examples/Receiver_Response.py

```diff
@@ -3,18 +3,22 @@
 
 Two files are required for the following example:
 
-    File1 contains the analytic signal acquired on the GDS oscilloscope
-    directly output from the AFG output. Each node pertains to signal with a
-    different frequency (in kHz) which are fit to a complex function to extract
-    the :math:`V_{p}` (in μV).
-    File2 contains the quadrature signal acquired on the receiver when the same
+file1
+    contains the analytic signal acquired on the (GW Instek GDS) oscilloscope
+    directly captured from the rf source (GW Instek AFG). Each node pertains to
+    signal with a different frequency (in kHz) which are fit to a complex
+    function to extract the :math:`V_{p}` (in μV).
+
+file2
+    contains the quadrature signal acquired on the receiver when the same
     signal of File1 is injected into it. Each node pertains to signal with a
     different frequency (in Hz) which are converted to a PSD. The amplitude (in
     dg) is calculated from the peak of the convolved PSD.
 
-    The receiver response is then the ratio of :math:`dg(\\nu)` to
-    :math:`μV(\\nu)` which is fit to an absolute sinc function.
+The receiver response is then the ratio of :math:`dg(\\nu)` to
+:math:`μV(\\nu)` which is fit to an absolute sinc function.
 """
+
```

Comment:
not sure why black didn't catch this but there should always be a space between the bottom of a docstring and the code